src/core/utils.js

```diff
@@ -4,7 +4,7 @@
 // As the name implies, this contains a ragtag gang of methods that just don't fit
 // anywhere else.
 import { lang as docLang } from "./l10n";
-import marked from "marked";
+import marked from "deps/marked";
```

Comment:
@saschanaz this was causing issues in the testing environment... but I think it will break in VSCode. Any suggestion for what I should set the path to?

Reply:
`"marked"` should just work as we set both requirejs and karma mappings. AFAIK this hasn't caused a problem recently...

Reply:
It's giving me grief via http://localhost:5000/tests/SpecRunner.html

Reply:
If "../deps/marked" is not too offensive, it seems to work across everything. Would it be ok to go with that?

Reply:
Actually, I think I know how to fix it.

Reply:
Anything except `"marked"` will break `@ts-check` (although `core/utils` currently lacks it). Maybe the SpecRunner should get the `paths` field synchronized with `profile-w3c-common.js`?